Rocket_Stabilization/RollPitchYaw/RollPitchYaw-udb5.X/nbproject/configurations.xml

```diff
@@ -143,7 +145,8 @@
         </subordinates>
       </compileType>
       <makeCustomizationType>
-        <makeCustomizationPreStepEnabled>true</makeCustomizationPreStepEnabled>
+        <makeCustomizationPreStepEnabled>false</makeCustomizationPreStepEnabled>
```

Comment:
@wjpremerlani  I think this change is what will fix the issue that you're experiencing.